src/calculateRentalCost.js

```diff
@@ -1,10 +1,18 @@
 /**
- * @param {number} days
+ * @param {number}
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sumOfDays = 40 * days;
+
+  if (days >= 7) {
+    sumOfDays -= 50;
```

Comment:
Instead of using an 'else if' after an 'if' with a 'return', you can use only 'if' with 'return' to simplify the logic and make the code cleaner.